packages/compass-components/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@mongodb-js/compass-components",
+  "version": "0.0.1",
+  "description": "React Components used in Compass",
+  "license": "SSPL",
+  "main": "lib/index.js",
+  "module": "./lib/.esm-wrapper.mjs",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib",
+    "package.json",
+    "README.md"
+  ],
+  "scripts": {
+    "check": "npm run lint && npm run depcheck",
+    "clean": "rimraf lib",
+    "precompile": "npm run clean",
+    "compile": "npm run compile-ts && gen-esm-wrapper . ./lib/esm-wrapper.mjs",
```

```suggestion
    "compile": "npm run compile-ts && gen-esm-wrapper . ./lib/.esm-wrapper.mjs",
```

to align with the `"module"` entry above

we should probably also add an
```js
  "exports": {
    "require": "./lib/index.js",
    "import": "./lib/esm-wrapper.mjs"
  }
```

section to the package.json, I don’t think Node.js understands the `"module":` key